api/src/db/DynamoUserDataClient.ts

```diff
@@ -141,6 +119,31 @@ export const DynamoUserDataClient = (
     return await search(statement);
   };
 
+  const getUsersWithBusinesses = async (
+    ExclusiveStartKey?: any
+  ): Promise<{ users: UserData[]; lastEvaluatedKey: any }> => {
```

Comment:
question, is lastEvaluatedKey type a string?

Reply:
No, `lastEvaluatedKey` is not a string. Its type is `Record<string, AttributeValue> | undefined`, which represents the key(s) used for continuing pagination in DynamoDB queries. I'm updating it throughout the codebase to ensure consistency and eliminate potential issues related to incorrect type handling.